### DISCUSSION TITLE
Build 관련 오류 .

### DISCUSSION THREAD

**Discussion by ChaMinXD** [TEAM-DAME/Dame-Android#128] [2. Q&A] [answered]
![image](https://user-images.githubusercontent.com/96709529/204992618-324e22e9-efb5-4870-80e1-6bf751a6657d.png)

![image](https://user-images.githubusercontent.com/96709529/204992656-62245322-e00f-478b-9dfb-89fda7e926d9.png)

빌드하려는데 이런 오류가 떠서 cache 삭제하고 빌드해도 안되네요 .. 무슨오류일까요 ? 혹시 해결방안 아시나해서 올려봅니다 ㅠㅠ

**Comment by onseok** [comment, marked as answer]
`Invalid caches / Restart` 를 한 번 더 해보시고, 안된다면, 
`루트 디렉토리의 .gradle 폴더를 삭제`하시고(.gitignore에 포함되어있음) 재시작해보시는 것을 추천드립니다.
